CHANGELOG.md

```diff
@@ -41,6 +41,10 @@ as necessary. Empty sections will not end in the release notes.
   specified explicitly in the JDBC URL.
   - `nessie.version.store.persist.jdbc.catalog`
   - `nessie.version.store.persist.jdbc.schema`
+- Support for token exchange in the Nessie client is deprecated and will be removed in a future 
```

Comment:
The token-exchange has actually been removed/replaced w/ the "proper" flows. Mind rewording this?

Reply:
I'd say _will be replaced_... we do not have a functional (alternative) token exchange flow ATM, do we?